raiden/transfermanager.py

```diff
@@ -68,11 +85,15 @@ def transfer_async(self, amount, identifier, target, callback=None):
             timeout.
         """
 
+        # Create a default identifier value
+        if not identifier:
```

Comment:
All right, funny thing flake8 did not warn me for this. Is there some way to configure it to warn me for such style intricacies?